lms/static/js/student_account/views/FormView.js

```diff
@@ -17,7 +17,13 @@
             formErrorsTpl: formErrorsTpl,
             formErrorsJsHook: 'js-form-errors',
             defaultFormErrorsTitle: gettext('An error occurred.'),
-            events: {},
+            events: {
```

Comment:
Kindly use [REDHOUSE_CUSTOM] comment whenever you do any change in edx's core code.